content/02.introduction.md

```diff
@@ -1,6 +1,19 @@
-#Introduction
-Set introduction for paper here
-Talk about problem, goal, and significance of paper
+# Introduction
+
+Populating knowledge bases is a daunting and arduous task that takes a significant amount of manual effort to accomplish.
```

Comment:
```suggestion
Populating knowledge bases often requires highly-trained scientists to read biomedical literature and summarize the results.
This manual curation process requires a significant amount of time and effort.
```

Reply:
I suggested a way to reframe these sentence, but the second sentence would really benefit from some specific figures provided by a reference/references if available.